pkg/trcapi/trctest/trc.go

```diff
@@ -7,31 +7,38 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/rvolosatovs/turtlitto/pkg/api"
+	"github.com/rvolosatovs/turtlitto/pkg/trcapi"
 	"go.uber.org/zap"
 )
 
-// Default handler for SetState messages, replying according to the API.
+// Default handler for state messages, replying according to the API.
```

Comment:
comment on exported function DefaultStateHandler should be of the form "DefaultStateHandler ..."